app/scripts/modules/kubernetes/src/v2/pipelines/stages/scaleManifest/scaleManifestConfig.html

```diff
@@ -1,7 +1,7 @@
 <div class="form-horizontal">
   <h4>Manifest</h4>
   <div class="horizontal-rule"></div>
-  <kubernetes-manifest-selector on-change="ctrl.handleManifestSelectorChange" selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
+  <kubernetes-manifest-selector modes='["static", "dynamic"]' on-change="ctrl.handleManifestSelectorChange" selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
```

Comment:
If you're using the `dynamic` mode you need to pass in the `application` object to the component, which could be more obvious.

Reply:
Will do, thanks